### PR TITLE
Added behind the scenes of browser

### PR DESCRIPTION
**Pull request by logeshpaul**

### CODE REVIEW

**Comment by logeshpaul** [comment]
@phildini waiting for your review.

File: README.rst [hunk 1]
```diff
@@ -476,6 +476,65 @@ The most common HTTPD servers are Apache for Linux, and IIS for windows.
 * The server will use PHP to interpret the index file, and catch the output.
 * The server will return the output, on the same request to the client.
 
+Behind the scenes of Browser
```
Comment (phildini) [inline README.rst]: *the* browser

File: README.rst [hunk 2]
```diff
@@ -476,6 +476,65 @@ The most common HTTPD servers are Apache for Linux, and IIS for windows.
...
+----------------------------------
+
+Once the server supplies the resources (HTML, CSS, JS, Image, etc.,)
+to browser it undergoes the below process
```
Comment (phildini) [inline README.rst]: the browser

File: README.rst [hunk 3]
```diff
@@ -476,6 +476,65 @@ The most common HTTPD servers are Apache for Linux, and IIS for windows.
...
+
+* Parsing - HTML, CSS, JS
+* Rendering - Construct DOM Tree → Render Tree → Layout of Render Tree →
+  Painting the render tree
+
+Browser
+-------
+
+The browser functionality is to present the web resource you choose, by
```
Comment (phildini) [inline README.rst]: browser's

File: README.rst [hunk 4]
```diff
@@ -476,6 +476,65 @@ The most common HTTPD servers are Apache for Linux, and IIS for windows.
...
+
+The way the browser interprets and displays HTML files is specified
+in the HTML and CSS specifications. These specifications are maintained
+by the W3C (World Wide Web Consortium) organization, which is the
+standards organization for the web.
+
+Browser user interfaces have a lot in common with each other. Among the
+common user interface elements are:
+
+* Address bar for inserting a URI
```
Comment (phildini) [inline README.rst]: An address bar...

File: README.rst [hunk 5]
```diff
@@ -519,7 +578,7 @@ set to "complete" and a "load" event will be fired.
 You can see the full algorithms for tokenization and tree construction
 in the HTML5 specification
 
-**Browsers' error tolerance**
+**Browsers error tolerance**
```
Comment (phildini) [inline README.rst]: I think Browser's is probably correct her

File: README.rst [hunk 6]
```diff
@@ -476,6 +476,65 @@ The most common HTTPD servers are Apache for Linux, and IIS for windows.
...
+* **The user interface:** this includes the address bar, back/forward
+  button, bookmarking menu, etc. Every part of the browser display except
+  the window where you see the requested page.
+* **The browser engine:** marshals actions between the UI and the rendering
+  engine.
+* **The rendering engine:** responsible for displaying requested content.
+  For example if the requested content is HTML, the rendering engine parses
+  HTML and CSS, and displays the parsed content on the screen.
+* **Networking:** for network calls such as HTTP requests, using different
+  implementations for different platform behind a platform-independent
```
Comment (phildini) [inline README.rst]: platforms

**Comment by phildini** [comment]
Thanks for contributing, and making the tests pass! I think the section could use a re-reading for:

- clarity 
- grammar
- quality of information 

Were it me, since it's a fair-sized section, I would get a friend to read it over and suggest edits.

**Comment by BenjaminProgram** [comment]
I am working on making these changes in my fork's pull request.

**Comment by logeshpaul** [comment]
@phildini Your comments seems valid, I will re-look into it & also wait for your complete review.

@BenjaminProgram thanks :+1:

**Comment by BenjaminProgram** [comment]
I fixed these issues in pull request #147

File: README.rst (hunk 5 above)
Comment (BenjaminProgram) [inline README.rst]: Yeah

**Comment by logeshpaul** [comment]
Thanks guys :)

**Comment by afzidkarim** [comment]
you welcome bro

**Pull request merged**